source/slang/slang-ir-layout.cpp

```diff
@@ -250,7 +250,7 @@ case kIROp_##TYPE##Type:                                        \
             resultLayout.size = align(resultLayout.size, fieldTypeLayout.alignment);
             resultLayout.alignment = std::max(resultLayout.alignment, fieldTypeLayout.alignment);
         }
-        *outSizeAndAlignment = rules->alignCompositeElement(resultLayout);
+        *outSizeAndAlignment = rules->alignCompositeElementOfNonAggregate(resultLayout);
```

Comment:
Tuple type is the same as struct type so this needs to be consistent with that case.